src/test/kotlin/uk/gov/justice/digital/hmpps/hmppstemplatepackagename/integration/wiremock/HmppsAuthMockServer.kt

```diff
@@ -20,12 +20,10 @@ class HmppsAuthApiExtension : BeforeAllCallback, AfterAllCallback, BeforeEachCal
 
   override fun beforeAll(context: ExtensionContext) {
     hmppsAuth.start()
-    hmppsAuth.stubGrantToken()
```

Comment:
don't think this should be part of a `beforeAll`, should be included when needed